sdk/keyvault/azure-keyvault-certificates/setup.py

```diff
@@ -78,7 +79,7 @@
             "azure.keyvault",
         ]
     ),
-    install_requires=["azure-core<2.0.0,>=1.0.0b5", "azure-common~=1.1", "msrest>=0.5.0"],
+    install_requires=["azure-core<2.0.0,>=1.0.0b2", "azure-common~=1.1", "msrest>=0.5.0"],
```

Comment:
Was the change from b5 to b2 intended?

Reply:
absolutely not, how the hell this happened here....